tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/descartes_tesseract_kinematics.h

```diff
@@ -0,0 +1,100 @@
+#ifndef DESCARTES_TESSERACT_KINEMATICS_H
+#define DESCARTES_TESSERACT_KINEMATICS_H
+
+#include <Eigen/Geometry>
+#include <vector>
+#include <descartes_light/interface/kinematics_interface.h>
+#include <descartes_light/utils.h>
+#include <tesseract_kinematics/core/forward_kinematics.h>
+#include <tesseract_kinematics/core/inverse_kinematics.h>
+
```

Comment:
Non tesseract headers need to be wrapped in the macro to prevent warnings.

Reply:
Done